src/core_modules/capture-core/components/WidgetEnrollment/MiniMap/MiniMap.component.tsx

```diff
@@ -1,24 +1,13 @@
-// @flow
 import React, { useState } from 'react';
 import { Map, TileLayer, Marker, Polygon } from 'react-leaflet';
 import { withStyles } from '@material-ui/core';
 import { dataElementTypes } from '../../../metaData';
 import { MapModal } from '../MapModal';
 import type { MiniMapProps } from './MiniMap.types';
+import { styles } from './MiniMap.types';
 import { convertToClientCoordinates } from './converters';
 import { useUpdateEnrollment } from '../dataMutation/dataMutation';
 
-const styles = () => ({
-    mapContainer: {
-        width: 150,
-        height: 120,
-    },
-    map: {
-        width: '100%',
-        height: '100%',
-    },
-});
-
```

Comment:
It looks like you moved the `styles` to src/core_modules/capture-core/components/WidgetEnrollment/MiniMap/MiniMap.types.ts to resolve the 'Cannot find name 'styles'' error. In another comment, I suggested a different approach to fix this issue. Once that's applied, you can move the styles back to MiniMap.component.tsx.